ui/View.elm

```diff
@@ -337,39 +348,35 @@ viewSourceDetails source =
             , tbody []
                 [ tr []
                     [ td [] [ text source.product ]
-                    , td [] [ text source.tree ]
+                    , td [] [ text <| Maybe.withDefault "" source.tree ]
```

Comment:
for the `record.build` above you use "unknown" as the default, do you want to use the same here?